homework.10/homework.10/circleViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  circleViewController.swift
+//  homework.10
+//
+//  Created by Stanislau on 03.04.2022.
+//
+
+import UIKit
+
+class circleViewController: UIViewController {
+    
+    let moveStep: CGFloat = 20
+    var toViewBorder: CGFloat = 0
```

Comment:
There's no point in having a common variable for all buttons because you don't use it anywhere else other than inside the funcitons. You can just create a local variable inside every function and it would be no difference